config/params/loader_test.go

```diff
@@ -38,6 +38,7 @@ var placeholderFields = []string{
 	"MAX_BLOB_COMMITMENTS_PER_BLOCK", // Compile time constant on BeaconBlockBodyDeneb.blob_kzg_commitments.
 	"MAX_BYTES_PER_TRANSACTION",      // Used for ssz of EL transactions. Unused in Prysm.
 	"MAX_EXTRA_DATA_BYTES",           // Compile time constant on ExecutionPayload.extra_data.
+	"MAX_PAYLOAD_SIZE",
```

Comment:
Is this supposed to be here?

Reply:
Shouldn't we handle this? The field was renamed from `GOSSIP_MAX_SIZE` to `MAX_PAYLOAD_SIZE`

Reply:
https://github.com/ethereum/consensus-specs/commit/454bd57cd0fd0ead7012b1ab81460f2fd7a5f49f

Reply:
I thought about this for a while. For background, this change is based on https://github.com/ethereum/consensus-specs/pull/4045. @nisdas is handling this change entirely in his upcoming PR, so I think it makes more sense to let him manage it all at once, it might get messy for him. In this PR, he will need to remove the variable from the placeholder anyway, or the test will fail